genesis.json

```diff
@@ -18126,7 +18126,20 @@
           "rewards": {
             "cumulative_reward_ratio": [
               {
-                "amount": "0.046434956961889469",
+                "amount": "0.003985610146904435",
```

Comment:
Here I had to push the rewards back one period, due to a panic of "negative coin amount" when calculating this validator's delegation reward